editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -1443,19 +1352,18 @@ function getCommentsOnExpression(sourceText: string, expression: TS.Node): Parse
   }
 }
 
-// FIXME It looks like this should now be merged with parseAttributeOtherJavaScript
-function parseJSExpression(
+export function parseAttributeOtherJavaScript(
```

Comment:
I think we should call this `parseJSExpressionMapOrOtherJavascript`, attributeOtherJavascript is kind of a legacy name which is misleading (because it is not just for attributes)

Reply:
Yeah good suggestion. I'll do that in another PR to save this one from becoming too noisy